components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -45,13 +45,15 @@
 #  define bfb_log(val) cxx_log(val)
 #  define bfb_log10(val) cxx_log10(val)
 #  define bfb_exp(val) cxx_exp(val)
+#  define bfb_sqrt(val) bfb_pow(val,0.5_rtype)
 #else
 #  define bfb_pow(base, exp) (base)**exp
 #  define bfb_cbrt(base) (base)**thrd
 #  define bfb_gamma(val) gamma(val)
 #  define bfb_log(val) log(val)
 #  define bfb_log10(val) log10(val)
 #  define bfb_exp(val) exp(val)
+#  define bfb_sqrt(val) sqrt(val)
```

Comment:
Not sure if `sqrt` is an intrinsic in F90. If it is, this line is great; if not, follow the example on line 51 of `#  define bfb_cbrt(base) (base)**thrd`, with `thrd` replaced by `half` in this case.

Reply:
From the info I can find, `sqrt` is an intrinsic function in Fortran.

Reply:
Yeah, google certainly thinks it's intrinsic, which gels with my memory. Going to leave this.